### ISSUE TITLE
Screen Readers cannot read the checkboxes

### ISSUE THREAD

**Issue by makatsis** [dipson88/treeselectjs#74]
Hello, 
is there a way a screen reader to have the ability to read the check boxes?

**Comment by makatsis** [comment]
This problem may be solved if you can add "id" at the checkboxes and "for" at the labels of the checkboxes.

**Comment by makatsis** [comment]
@dipson88  thank you for your answer 
What I want is to add at the label the attribute "for" and to the input the attribute "id"

For example, 
the following code the library gives
<div tabindex="-1" title="Bulgaria" class="treeselect-list__item treeselect-list__item--checked treeselect-list__item--focused" style="padding-left: 40px;" level="1" group="false"><div class="treeselect-list__item-checkbox-container"><span class="treeselect-list__item-checkbox-icon"><svg xmlns="http://www.w3.org/2000/svg" width="15" height="15" viewBox="0 0 25 25" fill="none" stroke="#000000" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"><polyline points="20 6 9 17 4 12"></polyline></svg></span><input tabindex="-1" type="checkbox" input-id="Bulgaria" class="treeselect-list__item-checkbox"></div><label class="treeselect-list__item-label">Bulgaria</label></div>

what I would like to get 
<div tabindex="-1" title="Bulgaria" class="treeselect-list__item treeselect-list__item--checked treeselect-list__item--focused" style="padding-left: 40px;" level="1" group="false"><div class="treeselect-list__item-checkbox-container"><span class="treeselect-list__item-checkbox-icon"><svg xmlns="http://www.w3.org/2000/svg" width="15" height="15" viewBox="0 0 25 25" fill="none" stroke="#000000" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"><polyline points="20 6 9 17 4 12"></polyline></svg></span><input tabindex="-1" type="checkbox" input-id="Bulgaria" **id="Bulgaria"** class="treeselect-list__item-checkbox"></div><label **for="Bulgaria"** class="treeselect-list__item-label">Bulgaria</label></div>

See the text in bold. 
Thank you!

**Comment by dipson88** [comment]
@makatsis I started work, but I faced a problem with Id. If we use Id/for like a name, we can't create a duplicated name. But the main goal was that we can create duplicated names but we should use unique values. I don't know how I can resolve this problem at this moment, maybe you can suggest something.
And the second question, can the reader read tabindex = -1? because these checkboxes have  tabindex = -1

**Comment by dipson88** [comment]
As I understand we can't read this checkbox, because we can't select them with the help of a keyboard. We use arrows, but we always focus on input with placeholders.

**Comment by makatsis** [comment]
And the second question, can the reader read tabindex = -1? because these checkboxes have tabindex = -1

@dipson88 you are right the screen reader cannot read the tabindex=-1, they skip it, and this might be the problem.
